main.tf

```diff
@@ -231,6 +231,22 @@ resource "google_compute_address" "static_nginx_ip" {
   }
 }
 
+resource "helm_release" "nginx_ingress_controller" {
+  # see https://kubernetes.github.io/ingress-nginx/deploy/#using-helm
+  # see https://kubernetes.github.io/ingress-nginx/deploy/#using-helm
```

Comment:
same comment twice :)

Reply:
your suggestion messed it up. LOL

Reply:
haha.... technically, your apply messed it up... haha